docs/ReleaseProcess.md

```diff
@@ -25,41 +27,67 @@ eBPF for Windows, and how to service it later.
     Version number updated to '0.12.0' in D:\work\ebpf-for-windows\scripts\..\installer\Product.wxs
     Rebuilding the solution, please wait...
     Regenerating the expected 'bpf2c' output...
-    Generating output for atomic_instruction_fetch_add.o
     ...
     ...
-    Generating output for printk_unsafe.o
     Expected 'bpf2c' output regenerated.
-    Please verify all the changes then submit the pull-request into the 'release/0.12' branch.
+    Please verify all the changes then submit the pull-request into the 'release/0.11' branch.
     ```
 1. Verify all the changes then commit all in the working branch.
     >NOTE: The formatting rules may complain about the formatting of the generated `.c` files from the script above. In this case, override them with the following (so they'll work with the `bpf2c_tests` verifying their content):
     >```bash
     >git commit --no-verify
     >```
-1. Create a **Draft** pull-request for your branch into the main repo's "`main`" branch (which you created in step 1), and title the PR as *"Release v`X.Y.Z`"* (replace "`X.Y.Z`" with the version number being released).
-1. Once the CI/CD pipeline for the PR completes, download the "`ebpf-for-windows - MSI installer (Build-x64_Release)`" and "`ebpf-for-windows - NuGet package (Build-x64_Release)`" build artifacts
-   (accessible via the "`Actions`" tab on GitHub).
-1. Extract the `*.nupkg` file, and rename it in the following format (replace "`X.Y.Z`" with the version number being released):
+1. Create a pull-request from your topic branch into the `main` branch of the original repo, with the title of the PR as *"Release v`X.Y.0`"* (replace "`X`" and "`Y`" with the version number being released).
+1. Once the PR is approved and merged into the "`main`" branch, of the original `ebpf-for-windows` repo, create a new release branch from `main` from the **previous PR's commit**, and name it "`release/X.Y.`".
+1. **IMPORTANT:** Once the release branch is created, no new feature work is allowed to be merged. However, bug fixes can be taken as deemed necessary by the maintainers and the release manager. Whenever applicable, these bug fixer should first be made in the main branch, and cherry-picked to the release branch. In case a bug-fix PRs are merged into the `release/X.Y` branch, the **latest commit** in the `release/X.Y` branch must be designated as the commit for the release.
+1. Follow the process in the [Release Branch Validation](ReleaseProcess.md#release-branch-validation) to ensure the quality of the release branch.
+1. In the triage meeting after successful creation of the release branch, the release manager must ask if any maintainer has any reasons to hold off the release. If not, move to the following steps.
```

Comment:
What happens if the follow sequence happens:
1 - Release branch created
2 - Release branch approved in triage meeting
3 - CI/CD Release validation fails.
4 - Bug fixes are required to pass CI/CD Release Validation, and they get merged.

Do we need a step 5 where re-approval/signoff is granted by maintainers? If so, does this happen in a subsequent triage meeting?

Reply:
No need for re-approval: the release approval is related to the feature it contains, and is only needed once. Bug fixes as a consequence of failures in the CICI Validation pipeline are by design.

Reply:
Agreed, You go by the `Release Branch Validation` process which does not require any approval process. I have updated line 44 to read "successful validation" instead of "creation".